editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Search/facets.ts

```diff
@@ -0,0 +1,29 @@
+import { IndexedNode } from "@opensystemslab/planx-core/types";
+import { FuseOptionKey } from "fuse.js";
+
+type SearchFacets = Array<FuseOptionKey<IndexedNode>>;
+
+const generalData: SearchFacets = ["data.fn", "data.val"];
+
+const fileUploadAndLabelData: SearchFacets = ["data.fileTypes.fn"];
+
+const calculateData: SearchFacets = [
+  "data.output",
+  {
+    name: "formula",
+    getFn: (node: IndexedNode) => Object.keys(node.data?.defaults || {}),
+  },
+];
+
+const listData: SearchFacets = [
+  "data.schema.fields.data.fn",
+  "data.schema.fields.data.options.data.val",
+];
+
+/** Data fields used across PlanX components */
+export const DATA_FACETS = [
+  ...generalData,
+  ...fileUploadAndLabelData,
+  ...calculateData,
+  ...listData,
+];
```

Comment:
Reminder `DrawBoundary` is another "special" case here I think ! 
```
const drawBoundaryData: SearchFacets = ["dataFieldBoundary", "dataFieldArea"];
```

Reply:
Very good catch thank you - missed this one!

Picked up in [bc27773](https://github.com/theopensystemslab/planx-new/pull/3576/commits/bc27773258a5482c38ce61c0006ce90fb0b01d06)